product-scenarios/1-manage-public-partner-private-apis/1.1-expose-service-as-rest-api-and-apply-qos/1.1.2-create-rest-api-by-import-oas-document/src/test/java/org/wso2/am/scenario/tests/rest/api/creation/RESTApiCreationUsingOASDocTestCase.java

```diff
@@ -49,8 +59,97 @@ public void init() throws APIManagerIntegrationTestException {
         }
         publisherURLHttp = "http://" + authority + ":9763/";
 
-        setKeyStoreProperties();
+        //setKeyStoreProperties();
```

Comment:
Any reason to comment out this?

Reply:
Yes, setKeystoreProperties is needed only in the place where handling with SSL(https endpoint calls). In this test case it is not need, hence removed. 
Fix : cd8f6d11c67af9a507d775a221e0c9a611bac79f